client/src/nv_ingest_client/message_clients/redis/redis_client.py

```diff
@@ -122,15 +125,36 @@ def ping(self) -> bool:
             return True
         except (RedisError, AttributeError):
             return False
+        
+    def fetch_message_morpheus(self, task_queue):
```

Comment:
The name here probably doesn't make sense. The purpose of adding this function was my realization that I could remove the other redis_client.py located at `nv-ingest/src/utils/redis` if this client had a method to simple pull messages from a queue so this was what I did for now.